src/frontend/Assets/Scripts/CSVReader.cs

```
@@ -55,20 +23,22 @@ public DataPoint(float time, float position)
         }
     }
 
-    private void LoadCSVData()
+    public List<DataPoint> LoadCSVData()
     {
+        List<DataPoint> dataPoints = new List<DataPoint>();
+    
         using (StreamReader reader = new StreamReader(csvPath))
         {
             // Read the header line
             string headerLine = reader.ReadLine();
-            if (headerLine == null) return;
+            if (headerLine == null) return dataPoints;
```

These should probably throw errors instead of returning an empty list